locale/ca/docs/index.md

```diff
@@ -43,7 +43,6 @@ La [secció d'ES6](/en/docs/es6/) descriu l'arbre dels grups de les característ
 característiques són activades per defecte en Node.js, juntament amb enllaços explicatius. També mostra com trobar
 quina versió de V8 fa servir una versió particular de Node.js.
 
-## Preguntes freqüents
+## Guides
```

Comment:
Why do you changed translation?

Reply:
Because there's no `/faq` link and I don't speak `ca`.